### ISSUE TITLE
morebits: lookupCreator callback doesn't run when there is no non-redirect version

### ISSUE THREAD

**Issue by Xi-Plus** [azatoth/twinkle#804]
Try this code on pages with no non-redirect version. The callback function doesn't run.
```javascript
var f = function(x) {
    console.log('done');
    console.log(x.getCreator());
}
var thispage = new Morebits.wiki.page(mw.config.get('wgPageName'));
thispage.setLookupNonRedirectCreator(true);
thispage.lookupCreation(f);
```
I guess it's due to
https://github.com/azatoth/twinkle/blob/d96fff1c6b82158683284a219b5ba80d217ebc96/morebits.js#L2809-L2816
Possible solution: `return` should be put in `if` statement at Line 2812.

**Comment by siddharthvp** [comment]
Yep yep, that's a bug, and that the solution. 

I found it and fixed it in my branch while working on #783, but yeah, it doesn't make sense to hold back this fix along with the much larger changes of that issue.

**Comment by Amorymeltzer** [comment]
Just made the fix from #805 live, thanks for the work you two!
